<launch>
	<!-- Trajectory Generation Specific Parameters -->
	<arg name="samplingTipMargin" 		default="4"  /> 
	<arg name="samplingOutMargin" 		default="12" /> 
	<arg name="samplingSpeedFactor" 	default="1.0" />		
	<arg name="enableHeadingSmoothing" 	default="false" />
	<arg name="startFromFrontAxel" 		default="false" />
	<arg name="enableForwardSimulation"	default="false" />
			
	<node pkg="op_local_planner" type="op_trajectory_generator" name="op_trajectory_generator" output="screen">
	
	<param name="samplingTipMargin" 		value="$(arg samplingTipMargin)"  /> 
	<param name="samplingOutMargin" 		value="$(arg samplingOutMargin)" /> 
	<param name="samplingSpeedFactor" 		value="$(arg samplingSpeedFactor)" />		
	<param name="enableHeadingSmoothing" 	value="$(arg enableHeadingSmoothing)" />
	<param name="startFromFrontAxel" 		value="$(arg startFromFrontAxel)" />
	<param name="enableForwardSimulation" 	value="$(arg enableForwardSimulation)" />
			
	</node>				
			
</launch>
